torch/_torch_docs.py

```diff
@@ -2360,6 +2360,10 @@ def merge_dicts(*dicts):
 corresponding to the polar coordinates with absolute value :attr:`abs` and angle
 :attr:`angle`.
 
+.. note::
+    This implementation follows C++'s `std::polar <https://en.cppreference.com/w/cpp/numeric/complex/polar>`_
+    which is different from Python's `cmath.polar` and SciPy's `linalg.polar`.
```

Comment:
How are they different?

Reply:
`cmath.polar` and  SciPy's `linalg.polar` are both the inverse of `torch.polar` (they in a complex number and returns `(r, theta)`)

Reply:
I will add a sentence about their differences in the doc.

Reply:
yeah btw I don't think we usually reference the std functions for the Python API so I am not sure if we should reference `std::polar` especially since the standard library functions don't work for complex numbers on CUDA (we call into thrust library!)

Reply:
When we're wrapping a standard library call it's not weird to reference it, like fmod:

https://pytorch.org/docs/master/generated/torch.fmod.html

The standard library is well documented.

As for CUDA, that's a good technical point but as long as the polar function we're calling there is supposed to be std::polar I think it's OK. As pointed out in your comment below, however, we don't actually call std::polar!

Reply:
> When we're wrapping a standard library call it's not weird to reference it, like fmod:

ahh ok!